next-tavla/src/Admin/components/DeleteModal/styles.module.css

```diff
@@ -0,0 +1,10 @@
+.closeButton {
+    position: absolute;
+    top: 10px;
+    right: 5px;
+    background: none;
+}
```

Comment:
Jeg liker ikke helt å sette posisjon på en så "hardkodet" måte. Finner du en annen måte å gjøre det på? Et tips er å se i `app/(admin)/components/Login/index.tsx` på hvordan det gjøres der

Reply:
`ml-auto` ga samme behavior 👨‍💻👍 takk for innspill 💯